docs/guides/lint-and-bundle.md

```diff
@@ -0,0 +1,57 @@
+# Lint and bundle API descriptions with Redocly CLI
+
+It's common to run more than one command on an API description during the API
+lifecycle. The most common combination is to `lint` an API description to check
```

Comment:
What's the rule of breaking the lines? 80 characters or less? As far as I'm aware, this project has no such Prettier requirement. It feels unnatural to break the sentences this way. Wouldn't it be better to put newlines at the end of a sentence?

Reply:
I just wanted to wrap so I went for 80 chars, we don't have rules for this. I can re-format to sentences.